tests/formik/Input.test.jsx

```diff
@@ -81,4 +81,14 @@ describe("formik/Input", () => {
       expect(screen.getByText("Invalid email address")).toBeInTheDocument()
     );
   });
+
+  it("should display validation error when string containing only whitespace is provided", async () => {
+    const onSubmit = jest.fn();
+    render(<TestForm onSubmit={onSubmit} />);
+    userEvent.type(screen.getByLabelText("First Name"), "    ");
+    userEvent.click(screen.getByText("Submit"));
+    await waitFor(() =>
+      expect(screen.getByText("First name is required")).toBeInTheDocument()
+    );
```

Comment:
```suggestion
    expect(await screen.findByText("First name is required")).toBeInTheDocument()
```

Reply:
Are you using `findByText` as suggested? I just verified and it works fine.

<img width="981" alt="image" src="https://github.com/bigbinary/neeto-ui/assets/22348659/edee41ab-c0aa-4b4c-91c8-3cb5a8a7aac7">